lib/shim/transaction-shim.js

```diff
@@ -89,7 +69,6 @@ TransactionShim.prototype.popTransactionName = popTransactionName
 TransactionShim.prototype.setTransactionName = setTransactionName
 TransactionShim.prototype.handleCATHeaders = handleCATHeaders
 TransactionShim.prototype.insertCATRequestHeaders = insertCATRequestHeaders
-TransactionShim.prototype.insertCATReplyHeader = insertCATReplyHeader
```

Comment:
We consider the shim a public API (it pains me for something called the 'shim' to be public but alas). As such, removing or renaming a function would be considered a breaking change and require a major release.

For now, would recommend leaving it in and having it be a pass-through for whatever function has replaced it.

Reply:
ah ok. i can restore w/ the tests